modules/public/avatars.go

```diff
@@ -0,0 +1,91 @@
+package public
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"code.gitea.io/gitea/modules/storage"
+
+	"gitea.com/macaron/macaron"
+)
+
+// AvatarOptions represents the available options to configure the AvatarHandler.
+type AvatarOptions struct {
+	Prefix      string
+	SkipLogging bool
```

Comment:
Is this necessary? The new logger system allows to set logging levels for different features individually.

Reply:
I followed the existing `Options` structure to handle this part. 
https://github.com/go-gitea/gitea/blob/d4cd4ed4422be222088dd8535a228b906f43fdc2/modules/public/public.go#L24-L33

Reply:
I didn't realize that.